src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+    private Figure figure;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getDefaultFigure() {
+        Figures[] values = Figures.values();
+        switch (values[random.nextInt(values.length)].toString()) {
+            case "CIRCLE":
+                figure = new Circle();
+                break;
+            case "ISOSCELES_TRAPEZOID":
+                figure = new IsoscelesTrapezoid();
+                break;
+            case "RECTANGLE":
+                figure = new Rectangle();
+                break;
+            case "RIGHT_TRIANGLE":
+                figure = new RightTriangle();
+                break;
+            case "SQUARE":
+                figure = new Square();
+                break;
+            default:
+        }
+        figure.setColor(colorSupplier.getRandomColor());
+        return figure;
```

Comment:
let's return here one specific default figure for example circle with default parameters)